locator/tablets.cc

```diff
@@ -165,6 +165,10 @@ std::ostream& operator<<(std::ostream& out, tablet_id id) {
     return out << size_t(id);
 }
 
+std::ostream& operator<<(std::ostream& out, global_tablet_id id) {
```

Comment:
Please have the changelog explain what a global tablet_id is. In what scope? node or cluster? Why is it needed?

Reply:
There is a comment above global_tablet_id class which explains the scope.

Reply:
Ok. The changelog should mention it too.